packages/zowe-explorer/src/Profiles.ts

```diff
@@ -96,11 +96,11 @@ export class Profiles extends ProfilesCache {
         const baseProfile = this.getBaseProfile();
 
         if (
-            !theProfile.profile.tokenType &&
-            (!theProfile.profile.user || !theProfile.profile.password) &&
-            baseProfile &&
-            baseProfile.profile.host !== theProfile.profile.host &&
-            baseProfile.profile.port !== theProfile.profile.port
+            (!theProfile.profile.tokenType && (!theProfile.profile.user || !theProfile.profile.password)) ||
+            ((theProfile.profile.user === "" || theProfile.profile.password === "") &&
```

Comment:
@jellypuno I still need to test this, but I was wondering since @katelynienaber's PR #1120 (Fixed errors with updating optional profile fields) is getting rid of empty strings in the profile, would this still work?

Reply:
@jellypuno Actually, I approved PR #1120 this morning. Do you maybe want to take a look at it and if you think it's good, we could go ahead and merge it into master and bring it into this branch?

Reply:
i'll do it right now 👍